Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Block Legacy Apps.json

```diff
@@ -28,7 +28,7 @@
       ]
     }
   },
-  "description": "Blocks apps trying to connect using legacy protocols (not Modern Authentication).",
+  "description": "What is it: Blocks legacy protocols (not Modern Authentication). \n Simeon Notes: Legacy protocols including POP, SMTP, IMAP, and MAPI can be used to bypass security controls in your environment. This policy does not apply to users in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/block-legacy-authentication",
```

Comment:
**What does this do?**
Blocks legacy protocols (POP, SMTP, IMAP, and MAPI) which can be used to bypass security controls in your environment. This policy does not apply to users in the Azure AD group 'Baseline - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. 

**Why should you use this?**
Improves security by prohibiting users from bypassing security controls in your environment.

**What is the end-user impact?**
Users may not use legacy protocols (POP, SMTP, IMAP, and MAPI).

**Learn more**
[How to: Block legacy authentication to Azure AD with Conditional Access](https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/block-legacy-authentication)